understanding/21/reflow.html

```diff
@@ -20,7 +20,7 @@ <h2>In brief</h2>
 		</section>
     <section id="intent">
         <h2>Intent of this Success Criterion</h2>
-        <p>The intent of this Success Criterion is to support people with low vision who need to enlarge text and read it in a single column.  When the browser zoom is used to scale content to 400%, it reflows - i.e., it is presented in one column so that scrolling in more than one direction is not necessary.</p>
+        <p>The intent of this success criterion is to support people with low vision who need to enlarge text and read it in a single column.  When the browser zoom is used to scale content to 400%, it reflows - i.e., it is presented in one column so that scrolling in more than one direction is not necessary.</p>
```

Comment:
```suggestion
        <p>The intent of this Success Criterion is to support people with low vision who need to enlarge text and read it in a single column.  When the browser zoom is used to scale content to 400%, it reflows - i.e., it is presented in one column so that scrolling in more than one direction is not necessary.</p>
```